.devcontainer/xdebug.ini

```diff
@@ -0,0 +1,7 @@
+zend_extension=xdebug.so
+xdebug.mode=debug,coverage
+xdebug.start_with_request=yes
+xdebug.client_host=host.docker.internal
+xdebug.client_port=9003
+xdebug.idekey=DBGP_IDEKEY
```

Comment:
I do not think that works. For the `DBGP_IDEKEY` environment variable to be use, I believe this line should be removed completely

```suggestion
```

Reply:
in my IDE (phpstorm) I can put any key as long as it's a string, it's the same with other IDEs like netbeans.

"Session ID Arbitrary name of the debugging session. The default is netbeans-xdebug. This value is important to note in cases where you need to set the xdebug.idekey property in php.ini, as in some remote debugging scenarios."

I can do a "sed" on the key value if you like.
it's better to think of the values as defaults, I'm looking to easily override them in another commit

Reply:
No need for `sed`, it should work with the environment variable after just deleting this line:

https://xdebug.org/docs/all_settings#idekey

> The default is based on the `DBGP_IDEKEY` environment setting